mapcompose/src/main/java/ovh/plrapps/mapcompose/api/MarkerApi.kt

```diff
@@ -203,15 +200,14 @@ fun MapState.onCalloutClick(cb: (id: String, x: Double, y: Double) -> Unit) {
  * @param id The id of the marker
  * @param deltaPx The displacement amount in pixels
  */
-fun MapState.moveMarkerBy(id: String, deltaPx: Offset) {
+fun MapState.moveMarkerBy(id: String, deltaPx: Point) {
```

Comment:
In this case, an `Offset` is more descriptive than `Point`.

Reply:
I'm not reviewing all the changes, because there are too many of them. Just came across these.